upstart/ua-premium.conf

```diff
@@ -0,0 +1,15 @@
+# ua-premium - Ubuntu Advantage Premium attach
+
+# This task is run on reboot to adapt to potential change in
+# kernels/environment across boot and update Ubuntu Advantage status
+# information.
+
+description    "Ubuntu Advantage premium attach"
+
+# Only start once cloud-init finishes detecting the DataSource on trusty
+start on file FILE=/var/lib/cloud/data/result.json
+
+task
+# Attempt to attach-premium action if machine is not already attached
```

Comment:
This doesn't look accurate; we run `attach-premium` whether or not the machine is already attached.

Reply:
updated to
# Attempt to attach-premium action. Does nothing if machine is already attached